src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -33,14 +24,949 @@ to make sure IE does not go into compatibility mode.
   level="warning"
 %}
 
-## Integrations
+&nbsp;
+## Integrating the SDK
+All our JavaScript-related SDKs provide the same API. Still, there are some differences between them, such as installation, which this section of the docs explains.
 
-All of our SDKs provide _Integrations_, which can be seen as some kind of plugins. All JavaScript SDKs provide default _Integrations_; please check details of a specific SDK to see which _Integrations_ it provides.
+**[Mimi note: Drop down here]**
 
-One thing is the same across all our JavaScript SDKs and that's how you add or remove _Integrations_, e.g.: for `@sentry/browser`.
+**[Mimi note: Dropdown which affects install instructions are shown (if nothing is provided in the URL, defaults to browser)]**
 
-### Adding an Integration
+- In the browser
+- Angular
+- Electron
+    - Includes some minidump explanation, maybe redirects to the native page
+- Ember
+- Node.js
+- React
+    - Includes a note pointing to React Native
+- Vue
+- Cordova
+- Ionic
+
+The quickest way to get started is to use the CDN hosted version of the JavaScript browser SDK:
+
+```
+<script src="https://browser.sentry-cdn.com/4.5.3/bundle.min.js" crossorigin="anonymous"></script>
+
+```
+
+&nbsp;
+### Connecting the SDK to Sentry
+After you've completed setting up a project in Sentry, Sentry will give you a value which we call a _DSN_ or _Data Source Name_. It looks a lot like a standard URL, but it’s just a representation of the configuration required by the Sentry SDKs. It consists of a few pieces, including the protocol, public key, the server address, and the project identifier.
+
+**[Mimi note: Drop down here]**
+
+You should `init` the Sentry Browser SDK as soon as possible during your page load:
+
+```
+Sentry.init({ dsn: 'https://<key>@sentry.io/<project>' });
+```
+
+Most SDKs will now automatically collect data if available; some require extra configuration as automatic error collecting is not possible due to platform limitations.
+
+&nbsp;
+## Capturing Errors
+### Capturing Errors/ Exceptions
+In JavaScript, you can pass an error object to `captureException()` to get it captured as an event.
+
+```
+try {
+  aFunctionThatMightFail();
+} catch (err) {
+  Sentry.captureException(err);
+}
+```
+
+{% capture __alert_content -%}
+It's possible to throw strings as errors. In this case, the Sentry SDK will not record tracebacks.
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Note"
+  content=__alert_content
+  level="warning"
+%}
+
+&nbsp;
+### Automatically Capturing Errors
+Sentry attaches global handlers to capture uncaught exceptions and unhandled rejections.
+
+**[Mimi note: Code Snippet]**
```

Comment:
We can go live with some missing code samples but this seems like a very important one.

Does it just work out of the box? If so the code can just be a "x=1/0" and a screenshot of how that looks?

Reply:
@billyvg will you help with this code snippet request?

Reply:
@MimiDumpling Hmm, do we want a code snippet of just bad code here?

Reply:
http://billy-sentry-sandbox--billyv.repl.co/
https://repl.it/@billyv/billy-Sentry-Sandbox?language=reactjs&folderId=196e878c-3a11-4dcd-b734-142c4b12cf4d
https://sentry.io/organizations/sentry-billy/issues/877249539/?query=is%3Aunresolved&statsPeriod=14d&utc=true

Reply:
https://sentry.io/share/issue/7d4eb8bc79c140fa88e49d61927925df/